### PR TITLE
add csa max wind change set and meanWindSpeed to load case

### PR DESCRIPTION
**Pull request by spidaDillon**

### CODE REVIEW

File: src/main/groovy/com/spidasoftware/schema/conversion/changeset/v10/CSAMaxWindLoadCaseChangeSet.groovy [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * ©2009-2023 SPIDAWEB LLC
+ */
+package com.spidasoftware.schema.conversion.changeset.v10
+
+import com.spidasoftware.schema.conversion.changeset.ConversionException
+import com.spidasoftware.schema.conversion.changeset.client.AbstractClientDataChangeSet
+import groovy.transform.CompileStatic
+
+/**
+ *
```
Comment (jasongarrett) [inline src/main/groovy/com/spidasoftware/schema/conversion/changeset/v10/CSAMaxWindLoadCaseChangeSet.groovy:11]: better comment

File: src/main/groovy/com/spidasoftware/schema/conversion/changeset/v10/CSAMaxWindLoadCaseChangeSet.groovy [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+        if (designJSON.containsKey("analysis")) {
+            List<Map> analysisList = designJSON.analysis as List<Map>
+            analysisList.removeAll { (it.analysisCaseDetails as Map).type == "CSA 2020 Maximum Wind" }
+        }
+    }
+
+    @Override
+    boolean revertResults(Map resultsJSON) throws ConversionException {
+        boolean anyChanged = false
```
Comment (jasongarrett) [inline src/main/groovy/com/spidasoftware/schema/conversion/changeset/v10/CSAMaxWindLoadCaseChangeSet.groovy:54]: anyChanged = super.revertResults(resultsJSON)

File: src/main/groovy/com/spidasoftware/schema/conversion/changeset/ConverterUtils.groovy [hunk 3]
```diff
@@ -96,6 +98,7 @@ class ConverterUtils {
         converter.addChangeSet(9, new EnvironmentClientDataChangeset())
         converter.addChangeSet(9, new ExtremeWindLoadCaseChangeset())
         converter.addChangeSet(9, new ClearancesChangeset())
```
Comment (jasongarrett) [inline src/main/groovy/com/spidasoftware/schema/conversion/changeset/ConverterUtils.groovy:100]: I bet IceDensityChangeSet should be here too.

**Review by jasongarrett — CHANGES_REQUESTED** [review]

File: src/main/groovy/com/spidasoftware/schema/conversion/changeset/v10/CSAMaxWindLoadCaseChangeSet.groovy (hunk 1 above)
Comment (spidaDillon) [reply src/main/groovy/com/spidasoftware/schema/conversion/changeset/v10/CSAMaxWindLoadCaseChangeSet.groovy:11]: Oops, I must have forgot to finish this

**Pull request merged**